File: Backend/Martins.Conciliacao.API/Code/Permissao/PermissaoSQL.vb

Namespace Code.Permissao
    Public Class PermissaoSQL
        Public Function SelectPermissao(NOMGRPACS As String) As String
      Return String.Format("select DISTINCT p.CODACSFUN, p.NOMACSFUN from CADGRPACSCLO g inner Join RLCGRPACS pg On g.CODIDTGRP=pg.CODIDTGRP inner Join CADACSSISCLO p On pg.CODACSCLO=p.CODACSCLO where g.NOMGRPACS In ({0})", NOMGRPACS)
    End Function
    End Class
End Namespace

File: Backend/Financeiro.ConciliacaoWebAPI/JsonWebToken/Code/Permissao/PermissaoSQL.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace JsonWebToken.Code.Permissao
{
  public class PermissaoSQL
  {
    public string SelectPermissao(string NOMGRPACS)
    {
      return string.Format(@"select DISTINCT p.CODACSFUN, p.NOMACSFUN from CADGRPACSCLO g
                                  inner Join RLCGRPACS pg on g.CODIDTGRP=pg.CODIDTGRP
                                  inner Join CADACSSISCLO p on pg.CODACSCLO=p.CODACSCLO
                                  where g.NOMGRPACS in ({0})", NOMGRPACS);
    }
  }
}
